src/codec.ts

```diff
@@ -30,6 +30,27 @@ export interface Field {
   value: Value;
 }
 
+export interface IProtoMessageParams {
+  value: object;
+  // fully qualified name of the proto representing the enum definition
+  fullName: string;
+  // Function representing the message with helper methods for deserializing
+  // and serializing messages. Method to serialize (i.e. "encode") should be
+  // present when writing data and method to deserialize (i.e. "decode") should
+  // be present while reading/querying data.
+  // User needs to pass typings/js generated by protobufjs.
```

Comment:
As someone with no experience with proto-columns, I have absolutely no idea what this means. Can we have a more concrete description, maybe with an example?